core/src/main/java/org/elasticsearch/search/aggregations/bucket/histogram/InternalDateHistogram.java

```diff
@@ -277,6 +277,31 @@ public String getWriteableName() {
         return Collections.unmodifiableList(buckets);
     }
 
+    /**
+     * Getter for testing only.
+     */
+    DocValueFormat getFormatter() {
+        return format;
+    }
+
+    /**
+     * Getter for testing only.
+     */
+    long getMinDocCount() {
+        return minDocCount;
+    }
+
+    /**
+     * Getter for testing only.
+     */
```

Comment:
I wonder if we need such javadoc comments. It's pretty self-explanatory that they are there for tests, those are the only classes that call them.